SwiftStaticLibraryTests/SwiftStaticLibraryTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  SwiftStaticLibraryTests.swift
+//  SwiftStaticLibraryTests
+//
+//  Created by Chad Landis on 8/7/17.
+//  Copyright © 2017 Chad Landis. All rights reserved.
+//
+
+import XCTest
+import SwiftStaticLibrary
+
+class SwiftStaticLibraryTests: XCTestCase {
```

Comment:
Had to manually create a unit test target for this (the Xcode project template for static library doesn't give you the option, but it is included for dynamic framework targets).  Once I setup the header paths for the unit test target, I was able to build and test.